postgresql/README.md

```diff
@@ -0,0 +1,314 @@
+
+    Name of user.
+
+Returns return code of executed expression.
+
+## postgresqlQuery
+
+Execute SQL query / psql internal command in db
+
+    postgresqlQuery schema query
+
+- schema
+
+    DB Schema (db name)
+
+- query
+
+    SQL query or internal command.
+
+Returns 0 on suxcess, print result to STDOUT.
+
+## postgresqlCreateDB
+
+Create new schema inherited from schema(template1 by default)
+
+    postgresqlQuery name [schema]
+
+- name
+
+    New db name
+
+- query
+
+    DB Schema (db name)
+
+Returns 0 on suxcess.
+
+## postgresqlDropDB
+
+Drop database
+
+    postgresqlDropDB name
+
+- name
+
+    Db name
+
+Returns 0 on suxcess.
+
+## postgresqlCreateLang
+
+Create new lang
+
+    postgresqlQuery name [schema]
+
+- name
+
+    language
+
+- schema
+
+    db name
+
+Returns 0 on suxcess.
+
+## postgresqlDropLang
+
+Drop language
+
+    postgresqlDropLang name
+
+- name
+
+    language
+
+- schema
+
+    Db name
+
+Returns 0 on suxcess.
+
+## postgresqlAddUser
+
+Add user with password.
+
+    postgresqlAddUser user password [schema]
+
+- user
+
+    User name.
+
+- password
+
+    Password for user.
+
+Returns 0 when user is successfully added, non-zero otherwise.
+
+## postgresqlDeleteUser
+
+Delete user.
+
+    postgresqlDeleteUser user [schema]
+
+- user
+
+    User name to delete.
+
+- schema
+
+    Schema name.
+
+Returns 0 when user is successfully deleted, non-zero otherwise.
+
+## postgresqlUDB
+
+Add user, DB and grant acces for user.
+
+    postgresqlUDB dbName user password
+
+- dbName
+
+    Name of DB which will be created.
+
+- user
+
+    User name to create.
+
+- password
+
+    User password.
+
+Returns 0 when all is sucefully created.
+
+## postgresqlPidFile
+
+Returns postgresql pid file.
+
+    postgresqlPidFile
+
+Returns path to postgresql pid file if exists.
+
+## postgresqlGetPid
+
+Prints PID of the postmaster.
+
+    postgresqlGetPid
+
+Prints process ID of the current parent postmaster process.
+Returns exit code 2 if the process is not running.
+
+## postgresqlLockFile
+
+Returns postgresql lock file.
+
+    postgresqlLockFile
+
+Returns path to postgresql lock file if exists.
+
+## postgresqlGetDataDir
+
+Returns path to postgresql data dir.
+
+    postgresqlGetDataDir
+
+Returns path to postgresql datadir (or where it may be).
+
+## postgresqlGetPort
+
+Returns port number where is/can be postgresql binded.
+
+    postgresqlGetPort
+
+Returns postgresql port number.
+
+## postgresqlChangeAuth
+
+Change authentification method for postgresql server.
+
+    postgresqlChangeAut <method>
+
+- method
+
+    Autentification method, which will be set for all users/hosts.
+
+Returns 0 on suxcess non zero in other cases.
+
+# AUTHORS
+
+- Branislav Blaskovic <bblaskov@redhat.com>
+Jakub Prokes <jprokes@redhat.com>
+Vacla Danek <vdanek@redhat.com>
+
+# POD ERRORS
+
+Hey! **The above document had some coding errors, which are explained below:**
+
+- Around line 213:
+
+    You forgot a '=back' before '=head2'
```

Comment:
Seems that you've documented some pod errors as well ;-)

Reply:
Good catch, thanks!

Reply:
Fixed.